integration-intygstjanst/src/test/java/se/inera/intyg/minaintyg/integration/intygstjanst/CertificateStatusServiceTest.java

```diff
@@ -105,10 +108,38 @@ void shouldNotIncludeStatusIfRelationsIsEmpty() {
   }
 
   @Test
-  void shouldNotIncludeSeveralStatuses() {
-    final var response = certificateStatusService.get(REPLACED_RELATIONS, SENT_RECIPIENT,
+  void shouldOnlyIncludeReplacedIfCertificateIsBothSentAndReplaced() {
```

Comment:
Are we missing a test? Something like... shouldOnlyIncludeReplacedIfCertificateIsBothNewAndReplaced